core/utils.py

```diff
@@ -0,0 +1,25 @@
+import jwt
+
+from functools     import wraps
+from django.http   import JsonResponse
+
+from zara.settings import SECRET_KEY, ALGORITHM
+from users.models  import User
+
+def signin_decorator(func):
+    @wraps(func)
+    def wrapper(self, request, *args, **kwargs):
+        try:
+            access_token = request.headers.get('Authorization', None)
+            payload      = jwt.decode(access_token, SECRET_KEY, ALGORITHM)
+            user         = User.objects.get(id = payload['id'])
+            request.user = user
+
+        except jwt.exceptions.DecodeError:
+            return JsonResponse({'message' : 'INVALID_TOKEN'}, status = 400)
+
+        except User.DoesNotExist:
+            return JsonResponse({'message' : 'INVALID_USER'}, status = 400)
```

Comment:
인증과 관련된 status code로 401을 사용합니다.
```suggestion
            return JsonResponse({'message' : 'INVALID_TOKEN'}, status = 401)

        except User.DoesNotExist:
            return JsonResponse({'message' : 'INVALID_USER'}, status = 401)
```

Reply:
확인 후 수정하겠습니다!